modules/manage/pages/manage-settings/configure-compact-settings.adoc

```diff
@@ -250,38 +250,32 @@ Formatted, this might appear as follows:
 ----
 {
   "autoCompactionSettings": {
-    "parallelDBAndViewCompaction": true,
-    "allowedTimePeriod": {
-      "fromHour": 0,
-      "toHour": 2,
-      "fromMinute": 0,
-      "toMinute": 0,
-      "abortOutside": false
-    },
+    "parallelDBAndViewCompaction": false,
+    "magmaFragmentationPercentage": 50,
     "databaseFragmentationThreshold": {
       "percentage": 30,
-      "size": 536870912
+      "size": "undefined"
```

Comment:
Were these taken from a run of couchbase server? I've not seen "undefined" returned before

Reply:
Mmm. 🤔 

Yes they were; I'll take another look.

Reply:
Fixed.